modules/ServerAPI/src/server/routes/AuthorRoutes.coffee

```diff
@@ -50,6 +50,14 @@ exports.allowedProjectsInternal = (user, callback) ->
 	csUtilities.getProjectStubsInternal (statusCode, allProjects) ->
 		config = require '../conf/compiled/conf.js'
 		_ = require 'underscore'
+
+		# Filter out any projects that exist in the config for filtering projects
+		projectFilterList = JSON.parse config.all.server.projects.filterList
+		allProjects = _.filter allProjects, (project, index) ->
+			console.log project
```

Comment:
Do we still need these console.log statements?

Reply:
No, I will remove.